lib/util/git/index.ts

```diff
@@ -628,7 +628,10 @@ export async function isBranchBehindBase(
   }
 }
 
-export async function isBranchModified(branchName: string): Promise<boolean> {
+export async function isBranchModified(
+  branchName: string,
+  baseBranch?: string,
```

Comment:
is there a use case where we don't pass the base branch?

Reply:
Good question! I meant to call that out in the PR description and forgot. 

I've pushed a commit that might clarify it -- I wasn't able to immediately come up with a base branch for the onboarding and migration flows that made a lot of sense, given I don't know much about them, but happy to workshop them and perhaps if there is a sensible choice we could make it required.